backend/climateconnect_main/settings.py

```diff
@@ -191,21 +194,23 @@
     AZURE_CONTAINER = env("AZURE_CONTAINER")
 
 STATIC_URL = (
-    "/static/"
-    if env("ENVIRONMENT") in ("development", "test")
-    else "https://"
+    "/static/" if env("ENVIRONMENT") in ("development", "test") else "https://"
     + env("AZURE_ACCOUNT_NAME")
     + "."
     + env("AZURE_HOST")
     + "/{}/".format(env("AZURE_CONTAINER"))
 )
+
+
 STATIC_ROOT = (
     env("STATIC_ROOT") if env("ENVIRONMENT") in ("development", "test") else "static/"
 )
 MEDIA_ROOT = env("MEDIA_ROOT")
 MEDIA_URL = "/media/"
 
 REST_KNOX = {"TOKEN_TTL": timedelta(days=120)}
+GDAL_LIBRARY_PATH = '/opt/homebrew/opt/gdal/lib/libgdal.dylib'
+GEOS_LIBRARY_PATH = '/opt/homebrew/opt/geos/lib/libgeos_c.dylib'
```

Comment:
Is this common for all mac and linux users?

Reply:
I'll remove these -- this was part of testing and I don't think we need them actually.